resources/kcp/charts/kyma-environment-broker/templates/deprovision-retrigger-job.yaml

```diff
@@ -78,19 +79,20 @@ spec:
                   readOnly: true
               {{- end}}
             {{- if and (eq .Values.global.database.embedded.enabled false) (eq .Values.global.database.cloudsqlproxy.enabled true)}}
-                - name: cloudsql-instance-credentials
```

Comment:
If workloadidentidy is disabled we won't mount the cloudsql-instance-credentials, this would still be needed if .Values.global.database.cloudsqlproxy.enabled and not .Values.global.database.cloudsqlproxy.workloadIdentity.enabled

Or I'm missing something? (applicable for all jobs as the pattern is the same).

Reply:
I strongly believe it's useless to mount the credentials there because the connection is established using cloudsql-proxy sidecar and credentials should only be required by it. I have performed a [similar cleanup for KEB](https://github.com/kyma-project/control-plane/pull/2956/files), which works fine after it, and think the jobs in this PR were reusing the pattern from KEB.

Reply:
They are mounted a bit lower, the mount I removed is not for cloudsql-proxy container. I strongly believe it's useless to mount the credentials there because the connection is established using cloudsql-proxy sidecar and credentials should only be required by it. I have performed a [similar cleanup for KEB](https://github.com/kyma-project/control-plane/pull/2956/files), which works fine after it, and think the jobs in this PR were reusing the pattern from KEB.